### PR TITLE
Changed default value for CMS_INTERNAL_IPS

### LINKED ISSUE #5344: Missing Toolbar + Admin dropdown is missing (Administration, Pages)
Upgrade from 3.2.5 to 3.3.0 and toolbar is gone in the frontend, the admin is missing all the options under Django Administration , I can only see logout and change password. I have reverted back to 3.2.5 and everything works as expected. I'm not sure what could be the issue.

Thanks in advanced.

Django 1.8.3
{% load cms_tags %} and {% cms_toolbar %} are in base.html

### PR DESCRIPTION
**Pull request by mkoistinen**
Fixes #5344.

### CODE REVIEW

**Comment by FinalAngel** [comment]
Changelog entry?

**Comment by yakky** [comment]
LGTM

**Comment by mkoistinen** [comment]
@FinalAngel is there something wrong with the one I provided, or did you overlook it?

File: CHANGELOG.txt [hunk 1]
```diff
@@ -20,6 +20,7 @@
 * Fixed typo in ``AdvancedSettingsForm`` error messages.
 * Fixed long standing bug that prevented apphook endspoints from being
   CSRF exempt.
+* Changed default value for CMS_INTERNAL_IPS.
```
Comment (czpython) [inline CHANGELOG.txt]: Add the `` to the setting.

File: docs/reference/configuration.rst [hunk 2]
```diff
@@ -1,4 +1,4 @@
-.. _configuration:
+NTER.. _configuration:
```
Comment (czpython) [inline docs/reference/configuration.rst]: typo?

**Comment by czpython** [comment]
Ok to merge

**Comment by FinalAngel** [comment]
👍  sorry totally overlooked it 😿

**Pull request merged**